app-crypt/swtpm/swtpm-0.10.0.ebuild

```diff
@@ -67,6 +67,7 @@ src_configure() {
 
 src_install() {
 	default
+	use test && rm -fr "${ED}/usr/libexec/installed-tests/swtpm"
```

Comment:
This still needs addressing (USE=test -> USE=test-install), or just always delete it.

Reply:
The directory and its contents are not present if without the "test" use flag. The tests were never intended to be installed into the system.

Reply:
OK. Please note that in the commit message.